static_inventories/build_library.yml

```diff
@@ -0,0 +1,27 @@
+---
+all:
+  children:
+    openstack_api:
+      hosts:
+        localhost:
+          ansible_python_interpreter: /usr/bin/env python
+    jumphost:     # build_servers
+      hosts:
+        vip_build:
```

Comment:
Using `vip_build` as jumphost does not make sense.

Reply:
I would assume:

1. ssh via jumphost
2. distribution of compiled apptainer image over http(s). 

If that is accaptable for the vip team, then we can use the security group for web servers; see this task in the openstack_networking role:
```
- name: "Create security groups for webservers in 'docs' or 'jenkins' inventory group."
```